src/core/bot_api/input.ts

```diff
@@ -16,10 +18,15 @@ export class InputImpl implements Input {
     }
   }
 
-  onText(regex: RegExp, handler: MessageHandler): void {
-    this.eventEmitter.on('text', (msg) => {
-      if (regex.test(msg.text)) {
-        handler(msg);
+  onText(regex: RegExp, handler: MessageHandler, errorHandler: ErrorHandler = (_, e) => { throw e; }): void {
```

Comment:
don't use errorHandler. This was a mistake. please use try { ... } catch in the plugin.

Reply:
meme

Reply:
can you revert this file?